enterprise/tpa_pipeline.py

```diff
@@ -79,6 +80,12 @@ def handle_enterprise_logistration(backend, user, **kwargs):
             handle_redirect_after_social_auth_login(backend, user)
         return
 
+    # With a successful SSO login, validate the enterprise customer's IDP config if it hasn't already been validated
+    enterprise_provider_config = enterprise_customer.identity_providers.filter(provider_id=sso_provider_id).first()
+    if not enterprise_provider_config.identity_provider.was_valid_at:
```

Comment:
hmm maybe I am just not reading the django right, but should these references be `enterprise_provider_config` or `enterprise_provider_config.identity_provider` ?

Reply:
the enterprise provider config is really just a reference to the provider config from platform. The validation lives in that provider config which we reference through the enterprise provider config

Reply:
ah ok maybe I just did not get the correct syntax etc, if this works locally we are all set!